src/app/portlets/dot-service-integration/dot-service-integration-configuration-list/dot-service-integration-configuration-list.component.scss

```diff
@@ -0,0 +1,124 @@
+@import "variables";
+
+:host {
+    background: $gray-bg;
+    box-shadow: $md-shadow-4;
+    display: flex;
+    height: 100%;
+    padding: $basic-padding * 3;
+}
+
+.dot-service-integration-configuration__header {
+    border-bottom: 1px solid $gray-light;
+    display: flex;
+
+    dot-avatar {
+        border-radius: 50%;
+        box-shadow: $md-shadow-2;
+        margin: 0 $basic-padding * 3 $basic-padding * 3 $basic-padding * 3;
+    }
+}
+
+.dot-service-integration-configuration__data {
+    font-size: $font-size-small;
+    width: 100%;
+}
+
+.dot-service-integration-configuration__service-name {
+    color: $black;
+    display: inline-block;
+    font-size: $font-size-xx-large;
+    font-weight: bold;
+}
+
+.dot-service-integration-configuration__service-key {
+    color: $gray;
+    display: inline-block;
+    font-size: $font-size-large;
+    height: 24px;
```

Comment:
$basic-padding * 3 ?

Reply:
no, because it's a fixed height and it should not be related with the padding of the site